src/mad_spark_alt/core/simple_qadi_orchestrator.py

```
@@ -338,8 +344,92 @@
+                if current_hypothesis.strip() and len(current_hypothesis.strip()) > 20:
+                    hypotheses.append(current_hypothesis.strip())
+                
+                # Additional fallback: try to extract content between common delimiters
+                if len(hypotheses) < min(self.num_hypotheses, 3):
+                    # Look for sections separated by double newlines
+                    sections = re.split(r'\n\s*\n', content)
+                    for section in sections:
+                        section = section.strip()
+                        if len(section) > 30 and len(hypotheses) < self.num_hypotheses:
+                            # Skip sections that look like headers or metadata
+                            if section.startswith("**Scale:**") or section.startswith("Scale:"):
```

Ruff (0.12.2)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)